docs/source/datasets.md

```diff
@@ -132,24 +150,45 @@ Note that mounting the dataset locally is only supported on Linux because it req
 Occasionally, scripts expect the input dataset at a fixed location, for example, data is always read from `/tmp/mnist`.
 AzureML has the capability to download/mount a dataset to such a fixed location. With the `hi-ml` package, you can
 trigger that behaviour via an additional option in the `DatasetConfig` objects:
+
 ```python
 from health_azure import DatasetConfig, submit_to_azure_if_needed
 input_dataset = DatasetConfig(name="my_folder",
                               datastore="my_datastore",
                               use_mounting=True,
                               target_folder="/tmp/mnist")
 run_info = submit_to_azure_if_needed(...,
-                                     input_datasets=[input_dataset])
+                                     input_datasets=[input_dataset],
+                                     strictly_aml_v1=True)
 # Input_folder will now be "/tmp/mnist"
 input_folder = run_info.input_datasets[0]
 ```
 
 This is also true when running locally - if `local_folder` is not specified and an AzureML workspace can be found, then the dataset will be downloaded or mounted to the `target_folder`.
 
+### Overwriting existing output datasets
+
+When creating an output dataset with the same name as an existing dataset, the default behaviour of `hi-ml` is to overwrite the existing dataset to avoid uploading errors. If you wish to disable this behaviour, it can be controlled using the `overwrite_existing` parameter:
```

Comment:
Not sure I understand "to avoid uploading errors"? You could add something that overwriting is helpful if you first run of a script fails, and you want to re-run.